vmm/Cargo.toml

```diff
@@ -28,6 +27,9 @@ rate_limiter = { path = "../rate_limiter" }
 seccomp = { path = "../seccomp" }
 sys_util = { path = "../sys_util" }
 
+[target.'cfg(target_arch = "x86_64")'.dependencies]
```

Comment:
and target_arch = "x86"

Reply:
All method labeled in this commit as x86_64 should be x86_64 and x86 as per KVM API documentation.

Reply:
Firecracker does not run on x86 out of the box. We have x86_64 specific code inside our implementation (dumbo crate).
I do not understand why you are invoking the KVM API documentation for the vmm crate toml file.  I did respect by hard the KVM API but for the kvm crate where we are implementing wrappers on the kvm api. Take a look at `ioctl_defs.h` and functions calling those ioctl (kvm/src/lib.rs).
By labeling this crate as `x86` we are also implying that it builds on x86, right? vmm depends on the `devices` crate which in turn depends on dumbo (`dumbo`
does not build on x86, take a look at `xor_rng_u32`).

Reply:
Since Firecracker isn't supposed to support `x86`, I think we should not have `cfg(target_arch = "x86")` anywhere in the code.

In the case of a crate that we can move to `crates.io`, it makes sense to also do `x86` as long as the crate is fully functional on `x86` as well.